bindings/wasm/README.md

```diff
@@ -0,0 +1,172 @@
+# IOTA Wallet Library - WebAssembly bindings
+
+WebAssembly (Wasm) bindings for TypeScript/JavaScript to the `wallet.rs` Wallet library.
+
+## Which bindings to choose?
+
+The `wallet.rs` Wallet library also offers dedicated [Node.js bindings](../nodejs). The differences with this package are outlined below.
+
+|               |   Wasm bindings   |   Node.js bindings    |
+|:--------------|:-----------------:|:---------------------:|
+| Environment   | Node.js, browsers |        Node.js        |
+| Installation  |         -         | Rust, Cargo required* |
+| Performance   |        ✔️          |          ✔️✔️           |
+| Proof-of-work |  Single-threaded  |    Multi-threaded     |
+| MQTT          |         ❌        |          ✔️            |
```

Comment:
This is not in the wallet, but we have rocksdb as storage for example which we don't have in wasm